pyproject.toml

```diff
@@ -27,7 +27,7 @@ openpyxl = { version = "^3.1.2", optional = true }
 pydataxm = { version = "^0.3.2", optional = true }
 ruamel-yaml = "^0.17.24"
 pycountry = { version = "^22.3.5", optional = true }
-ruff = "0.4.3"
+ruff = "~0.11"
```

Comment:
This is a bit risky though, it might change randomly and that will trigger all lockfiles to be updated in the monorepo which creates huge builds that takes forever to merge as they are updated every time someone merges to master.

Reply:
I think so. Considering it's locked in the precommit hook either way.